site/docs/components/badge/examples.mdx

```diff
@@ -36,14 +35,35 @@ Using a badge inline with text clarifies which part of the content the badge rel
 
 Specify the maximum displayed number with the `max` prop. If `value` exceeds `max`, it will clamp the value and append a (`+`) suffix.
 
-When you don't pass a max value to the component, the value will have a limitation of three digits by default, displaying 999+ for any values over 999.
+- Without a `max` value, the component limits the value to three digits by default, displaying `999+` for any values over 999.
 
 </LivePreview>
 <LivePreview componentName="badge" exampleName="BadgeWithString">
     
 ## Badge with string
 
-The `value` property can accept both a `number` and a `string`.
+The `value` property can accept both a `number` and a `string`. When you pass a string, the badge will not clamp the value.
+
+</LivePreview>
+
+<LivePreview componentName="badge" exampleName="DotBadge">
+
+## Dot badge
+
+A dot badge is a circular dot that notifies users of changes in state.
+
+### Best practices
+
+- Be consistent with the badge's position in a group of elements.
+- If there is a 3:1 contrast between the badge and the element it is attached to, the badge can be anchored to the element's contents.
+- If the icon graphic has number on the right top, don't anchor the badge directly to it to avoid blocking the number.
```

Comment:
Whilst this is okay, I think it is too generic @yunjungyeh. Our guidance is to avoid anchoring it to the content where the dot badge will obscure its meaning e.g. if it's an icon with a number and it gets covered. 

This is fine for now but we can re-visit with more general guidance.